tslint.json

```diff
@@ -4,7 +4,6 @@
     "align": false,
     "class-name": false,
     "comment-format": false,
-    "curly": false,
```

Comment:
I'd suggest we use: `"curly": [true, "ignore-same-line"],` instead of deleting this completely.

We might even want to consider the `as-needed` option, which discourages unnecessary braces.   If properly indented I think this is still a very readable style.


Either one could eliminate the need for many/most of the other changes in this commit.

Reply:
It appears that ignore-same-line is the default.

Reply:
:arrow_right: I find `ignore-same-line` generally unreadable compared to the expanded form, and even with Allman braces not worth the space savings. I corrected the ones I was already modifying, but I'll leave enforcement of the expanded form for a future pull request.

Reply:
After review, it seems I misinterpreted the available options. The style I prefer isn't fully supported, but the default (currently in use) is as close as tslint gets to it.